### PR TITLE
All tests in CoreTransformerTest have been reactivated and fixed.

### PR DESCRIPTION
**Pull request by oszoto**
# Changes

## CoreTransformerTest.scala

* Reactivated tests that were set to be ignored despite being functional.
* Fixed an interpreter issue in which WhileStmt would not exit after encountering an ExitStmt.
* Fixed tests that were not checking for the correct results.
* Reconstructed the following stmt files based on test code that referenced non existent modules: stmtForCore01.oberon; stmtCaseCore01.oberon; stmtCaseCore02.oberon; stmtCaseCore03.oberon; stmtCaseCore04.oberon.

**Aluno:** Gabriel Santos Menezes
**Matricula:** 17/0050157
**Grupo:** 4

### CODE REVIEW

**Comment by oszoto** [comment]
testes estão imprimindo um monte de valores; corrigindo

**Comment by oszoto** [comment]
# Changes
* Used setTestEnvironment() from Interpreter.scala to redirect all output stream to a null stream to all tests that required the stream.
* Deleted all instances of println within test code.

**Comment by rbonifacio** [comment]
thanks for these contributions, @oszoto. I have already integrated into the master branch.

File: src/test/scala/br/unb/cic/oberon/transformations/CoreTransformerTest.scala [hunk 1]
```diff
@@ -107,6 +108,7 @@ class CoreTransformerTest extends AnyFunSuite {
 
     val coreModule = coreVisitor.transformModule(module)
 
+    interpreter.setTestEnvironment
```
[the remark this answers is not in the document]
Comment (d-magit) [reply src/test/scala/br/unb/cic/oberon/transformations/CoreTransformerTest.scala:111]: Obs.: The same occurs later on line 737.

**Comment by d-magit** [comment]
I believe this PR is ready to be closed. Thanks for the contributions @oszoto, you did a really great job.

**Pull request merged**